src/openrct2/util/Util.cpp

```diff
@@ -590,6 +590,44 @@ money32 add_clamp_money32(money32 value, money32 value_to_add)
 
 #undef add_clamp_body
 
+uint8 lerp(uint8 a, uint8 b, float t)
+{
+    if (t <= 0)
+        return a;
+    if (t >= 1)
+        return b;
+
+    sint32 range  = b - a;
+    sint32 amount = (sint32)(range * t);
+    return (uint8)(a + amount);
+}
+
+float flerp(float a, float b, float t)
+{
+    if (t <= 0) return a;
+    if (t >= 1) return b;
+
+    float range = b - a;
+    float amount = range * t;
+    return a + amount;
+}
```

Comment:
`flerp` is only used in `lightfx.c`.

Reply:
I moved it here to not split lerp and flerp between two separate units.

Do you want me to move flerp back?

Reply:
I'm not bothered, just mentioned it in case you didn't realise.